erpnext_com/www/docs/user/manual/en/asset/fixed-asset-register.md

```diff
@@ -0,0 +1,12 @@
+<!-- add breadcrumbs -->
+# Fixed Asset Register
+
+Fixed asset register provides a unified view of all the details regarding current status of an asset. A Fixed asset register tracks all the fixed assets that the you have created. This not only includes assets that are currently in your possession, but also tracks those assets that are disposed.
```

Comment:
use relevant terminology.. disposed as in scrapped?

Reply:
Disposed include assets that are both scrapped and sold, I have described them below.